GG/src/Font.cpp

```diff
@@ -506,6 +506,8 @@ namespace {
             xpr::mark_tag whitespace_tag(4);
             xpr::mark_tag text_tag(5);
 
+            using std::placeholders::_1;
```

Comment:
Not needed, as you are addressing the placeholders with a qualified name.  So either drop this line or remove the namespace qualifier at their place of use.